org-roam-capture.el

```diff
@@ -41,6 +41,9 @@
 (defvar org-roam-directory)
 (defvar org-roam-mode)
 (defvar org-roam-title-to-slug-function)
+(defvar org-roam-dailies-capture--file-name-default)
```

Comment:
let's remove these, and throw an error if the template doesn't contain it.

Reply:
Will be addressed in another PR.